{
    "data": [
        {
            "test_name": "mantidqt_qt5.test_sliceviewer_presenter.test_sliceviewer_presenter",
            "latest_failed_build": "3681",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 3,
            "past_outcome": {
                "build_keys": [
                    "3682",
                    "3681",
                    "3680",
                    "3679",
                    "3678",
                    "3677",
                    "3676",
                    "3675",
                    "3674",
                    "3673",
                    "3672",
                    "3671",
                    "3670",
                    "3669",
                    "3668",
                    "3667",
                    "3666",
                    "3665",
                    "3664",
                    "3663",
                    "3662",
                    "3661",
                    "3660",
                    "3659",
                    "3658",
                    "3657",
                    "3656",
                    "3655",
                    "3654",
                    "3653"
                ],
                "past_outcome_list": [
                    "None",
                    "Failed",
                    "Failed",
                    "Failed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "mantidqt_qt5.test_sliceviewer_presenter.test_sliceviewer_presenter",
                "build": "3681",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "7.84",
                "stacktrace": "QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to &#x27;/tmp/runtime-abc&#x27;\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nlibEGL warning: DRI2: failed to open swrast (search paths /usr/lib64/dri)\nQXcbIntegration: Cannot create platform OpenGL context, neither GLX nor EGL are enabled\n...........................................E..\n======================================================================\nERROR: test_y_axes_editor_calls_dimensions_changed (test_sliceviewer_presenter.SliceViewerTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_presenter.py&quot;, line 744, in test_y_axes_editor_calls_dimensions_changed\n    yaxes_edit = SliceViewYAxisEditor(\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/python/mantidqt/mantidqt/widgets/sliceviewer/presenters/presenter.py&quot;, line 598, in __init__\n    super(SliceViewYAxisEditor, self).__init__(canvas, axes)\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/applications/workbench/workbench/plotting/propertiesdialog.py&quot;, line 249, in __init__\n    self.create_model()\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/qt/applications/workbench/workbench/plotting/propertiesdialog.py&quot;, line 179, in create_model\n    memento.min, memento.max = getattr(self.axes, &quot;get_{}lim&quot;.format(self.axis_id))()\nValueError: not enough values to unpack (expected 2, got 0)\n\n----------------------------------------------------------------------\nRan 46 tests in 4.024s\n\nFAILED (errors=1)\n\n          Start   48: KernelTest_FloatingPointComparisonTest\n"
            }
        },
        {
            "test_name": "python.api.RunTest.RunTest",
            "latest_failed_build": "3678",
            "agent_name": "Linux",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-linux",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "3682",
                    "3681",
                    "3680",
                    "3679",
                    "3678",
                    "3677",
                    "3676",
                    "3675",
                    "3674",
                    "3673",
                    "3672",
                    "3671",
                    "3670",
                    "3669",
                    "3668",
                    "3667",
                    "3666",
                    "3665",
                    "3664",
                    "3663",
                    "3662",
                    "3661",
                    "3660",
                    "3659",
                    "3658",
                    "3657",
                    "3656",
                    "3655",
                    "3654",
                    "3653"
                ],
                "past_outcome_list": [
                    "None",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "None",
                    "None",
                    "None",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.api.RunTest.RunTest",
                "build": "3678",
                "agent_name": "Linux",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "0.66",
                "stacktrace": ".......E.........\n======================================================================\nERROR: test_get_time_roi (RunTest.RunTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/jenkins_workdir/workspace/pull_requests-conda-linux/Framework/PythonInterface/test/python/mantid/api/RunTest.py&quot;, line 48, in test_get_time_roi\n    values = roi.toSplitters()\nAttributeError: &#x27;TimeROI&#x27; object has no attribute &#x27;toSplitters&#x27;\n\n----------------------------------------------------------------------\nRan 17 tests in 0.005s\n\nFAILED (errors=1)\n\n          Start  137: KernelTest_UtilsTest\n"
            }
        }
    ]
}